README.md

```diff
@@ -18,6 +18,8 @@ This is a plugin for Homebridge that allows you to control your RemoteNow-enable
 - Homebridge 1.3.0 or later.
 - Python 3.8 with [paho-mqtt](https://pypi.org/project/paho-mqtt/) version 1.6.1 and [netifaces](https://pypi.org/project/netifaces/).
 - A Hisense TV that supports the RemoteNow app ([App Store](https://apps.apple.com/us/app/remotenow/id1301866548) or [Play Store](https://play.google.com/store/apps/details?id=com.universal.remote.ms&hl=en&gl=US)).
+  - WakeOnLan (WOL) must be enabled on the TV to turn it on with this plugin.
+  - fixed to a static IP address or a static DHCP reservation.
```

Comment:
Does this sentence refer to the TV or the WOL configuration?

If it's the former, I'd use:

```suggestion
  - The TV must be configured with static IP Address or a static DHCP reservation.
```